lib/_tls_wrap.js

```diff
@@ -171,10 +179,14 @@ function oncertcb(info) {
       if (!self._handle)
         return self.destroy(new Error('Socket is closed'));
 
-      try {
-        self._handle.certCbDone();
-      } catch (e) {
-        self.destroy(e);
+      if (isLibreSSL) {
+        self._handle.endParser();
```

Comment:
This is not quite right, a call to certCbDone() is probably appropriate here somewhere too, original iteration completely removed certCbDone when LibreSSL but this current one only removes pieces of it so it's still usable.